tests/sentry/ingest/test_ingest_consumer.py

```diff
@@ -40,7 +42,7 @@ def _get_test_message(project):
         "start_time": time.time(),
         "event_id": event_id,
         "project_id": 1,
-        "payload": normalized_event,
+        "payload": six.binary_type(json.dumps(normalized_event)),
```

Comment:
will not work in py3 , six.binary_type translates to bytes which needs a encoding="..." parameter